### PR TITLE
style(ui-contract-editor): Drag-Drop Handle Resize - I137

### LINKED ISSUE #137: Reduce height of drag and drop handle so short clauses don't have large space
### Feature Request 🛍️
Clauses with 1-3 lines of text have an awkward space at the bottom caused by the drag and drop handle. We can mitigate this by compressing the drag and drop handle.

the drag and drop buttons in the UI are spaced further apart than they are in the mockup. Making the buttons closer together like in the mockup should mitigate some of this issue.
image

mockup:
image

PM me for invision link to mockup

reducing this space only on very small clauses could help
image

### PR DESCRIPTION
**Pull request by K-Kumar-01**
Signed-off-by: k-kumar-01 <kushalkumargupta4@gmail.com>

# Closes #137 <CORRESPONDING ISSUE NUMBER>
Styling changes applied to the drag & drop handle so that short clauses don't have large space.

### Changes

- <ONE> Changes in margin
- <TWO> Some changes in svg and height

### Screenshots or Video

Screenshot from 2021-03-03 00-37-03

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`
- [ ] Manual accessibility test performed
    - [ ] Keyboard-only access, including forms
    - [ ] Contrast at least WCAG Level A
    - [ ] Appropriate labels, alt text, and instructions

### CODE REVIEW

File: packages/ui-contract-editor/src/components/Clause/index.js
Comment (DianaLease) [inline packages/ui-contract-editor/src/components/Clause/index.js:161]: Instead of adding these styles inline, I would add them to the styled component here: https://github.com/accordproject/web-components/blob/7ea0cb5e35c11fd45e786f3a56ea39854aa05ed0/packages/ui-contract-editor/src/components/styles.js#L146

**Review by DianaLease — COMMENTED** [review]
Looks good! Added one minor comment.

File: packages/ui-contract-editor/src/components/Clause/index.js
Comment (K-Kumar-01) [reply packages/ui-contract-editor/src/components/Clause/index.js]: @DianaLease 
I have removed the inline styles and converted them to styled-components. Let me know if there are any more changes required. :)

**Review by Michael-Grover — CHANGES_REQUESTED** [review]
Hi @K-Kumar-01, this looks great! I just have a few comments:

It looks like there are 8 circles in the code, but only 6 are visible. Can we delete the two extra ones in the code?
image

it looks like the distance between the dots and the bottom arrow is greater than the distance between the dots and the top arrow. I think we should make these distances the same.
image

It looks like the bottom arrow is now colliding with the clause template's outline at the bottom. I think we should move the buttons up to be vertically centered inside the space (I outlined this as a red box in the last image here)
image
image

**Comment by K-Kumar-01** [comment]
@Michael-Grover 
> It looks like there are 8 circles in the code, but only 6 are visible. Can we delete the two extra ones in the code?

I tried deleting them but it caused some styling issues initially. Also since they were not hampering with styles when left untouched I let them be there


> it looks like the distance between the dots and the bottom arrow is greater than the distance between the dots and the top arrow. I think we should make these distances the same.

This might cause some increase in the space


> It looks like the bottom arrow is now colliding with the clause template's outline at the bottom. I think we should move the buttons up to be vertically centered inside the space 

I will try to see if I can find a solution to this

**Comment by K-Kumar-01** [comment]
@Michael-Grover 
I changed some more styles. 
Screenshot from 2021-03-04 01-04-50

Zoomed.
Screenshot from 2021-03-04 01-07-04

Are position and arrow correct now? (Yellow background only for testing purpose)

Also, the dots are causing an issue when I remove the code. So can it be kept as it is?

**Comment by Michael-Grover** [comment]
@K-Kumar-01 this looks good to me, thanks. This change isn't in Netlify yet, right? I looked at Netlify and the arrow is still intersecting with the clause template border

image

**Comment by K-Kumar-01** [comment]
> @K-Kumar-01 this looks good to me, thanks. This change isn't in Netlify yet, right? I looked at Netlify and the arrow is still intersecting with the clause template border
> 
> image

No I have not yet pushed the code. I wanted a review so I just shared images. I will make a pr ASAP.

**Comment by K-Kumar-01** [comment]
@Michael-Grover 
I have made the changes in the PR. Let me know if there are any changes required.

**Comment by Michael-Grover** [comment]
@DianaLease 
> Looks good! Added one minor comment.

I can't find the comment you added. Is there anything you think should be changed?


@K-Kumar-01 it looks great to me, thanks! I think we should wait for @DianaLease or another engineer's review to merge.

**Review by Michael-Grover — APPROVED** [review]
lgtm, thanks!

**Comment by Michael-Grover** [comment]
@DianaLease never mind, I see your comment was resolved. It looks like this PR still requires an engineer approval before it can be merged though.

**Review by irmerk — APPROVED** [review]
👏🏻

**Pull request merged**
